components/blocks/youTube.js

```diff
@@ -1,42 +1,41 @@
 import React from "react";
 
-export default class YouTube extends React.Component {
-  render() {
-    const props = this.props;
-    let YouTubeBlock;
+const YouTube = ({ caption, video_id }) => {
```

Comment:
video_id -> videoID

(it would be great to somehow rename all these variables automatically... not sure if there's a tool for that.)

Reply:
Fixed on 051f9fe. Will look into some tools for that!